src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,328 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static Task[] taskList = new Task[100];
+    static int taskCount=0;
```

Comment:
According to the Java coding standard, there should be white spaces around the equal sign.